js/geo/lng_lat.js

```diff
@@ -18,6 +18,9 @@ var wrap = require('../util/util').wrap;
  * @param {number} lat Latitude, measured in degrees.
  * @example
  * var ll = new mapboxgl.LngLat(-73.9749, 40.7736);
+ * @see {@link https://www.mapbox.com/mapbox-gl-js/example/mouse-position/|Get coordinates of the mouse pointer}
```

Comment:
We have a global `baseurl` config value that can be used for this purpose. 

 - [definition for production site](http://github.com/mapbox/mapbox-gl-js/blob/master/_config.mb-pages.yml#L6)
 - [definition for dev site](http://github.com/mapbox/mapbox-gl-js/blob/master/_config.yml#L6)
 - [example usage](http://github.com/mapbox/mapbox-gl-js/blob/master/docs/_layouts/pages.html#L8-L9)